### PR TITLE
Add custom domain option under links

### PR DESCRIPTION
**Pull request by meyskens**
Add the ability to use your own domain with Cast. 
schermafbeelding 2017-01-12 om 14 36 03
schermafbeelding 2017-01-12 om 14 36 16

*Do not merge before ITFrame's PR*

### CODE REVIEW

File: src/manage/cast/links/links.html [hunk 1]
```diff
@@ -53,24 +69,77 @@ <h4 class="list-group-item-heading">Cast</h4>
...
+            <label for="custom-domain">Your subdomain</label>
+            <div class="input-group">
+                <span class="input-group-addon">https://</span>
+                <input type="text" class="form-control" id="custom-domain" ng-model="ctrl.customDomain">
+            </div>
+            <br>
+            <p>Please set the following DNS record on your domain:</p>
+            <table class="table">
+                <tr>
+                    <th>name</th>
```
Comment (leoetlino) [inline src/manage/cast/links/links.html]: Name

File: src/manage/cast/links/links.html [hunk 2]
```diff
@@ -53,24 +69,77 @@ <h4 class="list-group-item-heading">Cast</h4>
...
+                    <th>type</th>
```
Comment (leoetlino) [inline src/manage/cast/links/links.html]: Please capitalise this and the below header

File: src/manage/cast/links/links.html [hunk 3]
```diff
@@ -53,24 +69,77 @@ <h4 class="list-group-item-heading">Cast</h4>
...
+                <col style="width: 30%">
+            </colgroup>
+            <div class="row">
+                <div class="col-sm-3 pull-right">
+                    <button ng-click="ctrl.setCustomDomain()" class="btn btn-block btn-primary" ng-disabled="!ctrl.isValidDomain(ctrl.customDomain) || ctrl.isSaving || ctrl.customDomain === ctrl.config.hostname.replace('https://', '')"><i class="fa fa-save"></i> Save</button>
+                </div>
+            </div>
+        </li>
+    </ul>
+</section>
```
Comment (leoetlino) [inline src/manage/cast/links/links.html]: Missing newline

File: src/manage/cast/links/links.html (hunk 2 above)
Comment (meyskens) [inline src/manage/cast/links/links.html]: `<thead>`?

**Pull request merged**